### PR TITLE
Events

### PR DESCRIPTION
**Pull request by John-Arboleda**
# Events

"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
- Make simple calculations.
- Read a random math-related quote.

### Learning Objectives
- [x] Handle events in a React app.
- [x] Use React life cycle methods.
- [x] Understand the mechanism of lifting state up.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,41 +1,63 @@
...
+        <button className="number3 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '3')); }}>3</button>
+        <button className="number4 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '4')); }}>4</button>
+        <button className="number5 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '5')); }}>5</button>
+        <button className="number6 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '6')); }}>6</button>
+        <button className="number7 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '7')); }}>7</button>
+        <button className="number8 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '8')); }}>8</button>
+        <button className="number9 numbers" type="button" onClick={() => { this.handleStateChange(calculate(this.state, '9')); }}>9</button>
+        <button className="buttonAc symbols" type="button" onClick={() => { this.handleStateChange(calculate(this.state, 'AC')); }}>AC</button>
```
Comment (enaburekhan) [inline src/components/calculator.js:52]: - [ ] Nice job here. I would like to point out here that the highlighted `AC` button is currently not working. I would suggest you recheck the code implementation in the `calculate.js`. I'm sure its function is to clear the screen.

File: src/components/calculator.js [hunk 2]
```diff
@@ -1,41 +1,63 @@
...
     return (
       <div className="calculator">
         <div className="resultShow">
-          <p className="resultNumber">{ result }</p>
+          <p className="resultNumber">
+            { total }
+            { ' ' }
+            { operation }
+            { ' ' }
+            { next }
```
Comment (enaburekhan) [inline src/components/calculator.js:39]: - [ ] The total value displayed on the screen, should not include the `operation` symbol. I will suggest you remove it.

image

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi John-Arboleda, 

Good job so far!

- [ ] Major highlights

- The PR summary and commit messages are very descriptive.
- The codebase is clean, easy to read and understand.
- Passed all linters checks

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] I will suggest you include how to run the application locally in your readme. You can achieve this in your getting started section.

I had to add this comment here because the readme was not part of the below review

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by oloomoses — APPROVED** [review]
Hi @John-Arboleda, 

Well done on making the suggested changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oloomoses** [comment]
Kindly reach dm on slack

**Pull request merged**